gradle/spec/fixtures/buildfiles/.gitignore

```diff
@@ -0,0 +1 @@
+/.gradle/
```

Comment:
Why?
It's useful to invoke gradle against these files to validate they are valid:
```
$ gradle -b custom_repos_build_assignment_uri.gradle tasks

> Task :tasks

------------------------------------------------------------
Tasks runnable from root project 'buildfiles'
------------------------------------------------------------

Build tasks
-----------
assemble - Assembles the outputs of this project.
build - Assembles and tests this project.
buildDependents - Assembles and tests this project and all projects that depend on it.
buildNeeded - Assembles and tests this project and all projects it depends on.
classes - Assembles main classes.
clean - Deletes the build directory.
jar - Assembles a jar archive containing the main classes.
testClasses - Assembles test classes.

Build Setup tasks
-----------------
init - Initializes a new Gradle build.
wrapper - Generates Gradle wrapper files.

Documentation tasks
-------------------
javadoc - Generates Javadoc API documentation for the main source code.

Help tasks
----------
buildEnvironment - Displays all buildscript dependencies declared in root project 'buildfiles'.
dependencies - Displays all dependencies declared in root project 'buildfiles'.
dependencyInsight - Displays the insight into a specific dependency in root project 'buildfiles'.
help - Displays a help message.
javaToolchains - Displays the detected java toolchains.
outgoingVariants - Displays the outgoing variants of root project 'buildfiles'.
projects - Displays the sub-projects of root project 'buildfiles'.
properties - Displays the properties of root project 'buildfiles'.
tasks - Displays the tasks runnable from root project 'buildfiles'.

Verification tasks
------------------
check - Runs all checks.
test - Runs the unit tests.

Rules
-----
Pattern: clean<TaskName>: Cleans the output files of a task.
Pattern: build<ConfigurationName>: Assembles the artifacts of a configuration.

To see all tasks and more detail, run gradle tasks --all

To see more detail about a task, run gradle help --task <task>

BUILD SUCCESSFUL in 698ms
1 actionable task: 1 executed



$ gradle -b basic_build.gradle tasks

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/thepwagner/git/github/dependabot-core/gradle/spec/fixtures/buildfiles/basic_build.gradle' line: 3

* What went wrong:
A problem occurred evaluating root project 'buildfiles'.
> Plugin with id 'com.github.johnrengelman.shadow' not found.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 651ms
```